.contracts.json

```diff
@@ -19,5 +19,8 @@
   },
   "arbitrumRinkebyTestnet": {
     "router": "0xb5a35165047fed7440D3a75909c0949bf1943696"
+  },
+  "arbitrumGoerliTestnet": {
+    "router": "0xe82c4D8b993D613a28600B953e91A3A93Ae69Fd6"
```

Comment:
We can verify this deployment after 1:30PM ET later today.

Reply:
Just verified it: https://goerli-rollup-explorer.arbitrum.io/address/0xe82c4D8b993D613a28600B953e91A3A93Ae69Fd6/contracts

Reply:
Thank you!